### DISCUSSION TITLE
Freetz linux 1.4.1 make not working

### DISCUSSION THREAD

**Discussion by redboyhun** [Freetz-NG/freetz-ng#1035] [Allgemein] [answered]
Tried these guides:
https://www.ip-phone-forum.de/threads/buildumgebung-freetz-linux.199449/#post-1400234
https://freetz-ng.github.io/freetz-ng/wiki/10_Beginner/newbie.de.html

but it is not working, I get make too old error:
root@freetz-linux:/home/freetz/freetz-ng# make
Freetz-NG 24360-1b852e4dc1 master 2024-08-30
Makefile:151: *** Your make (3.81) is too old. Go get at least 3.82.  Stop.
make[1]: *** [envira] Beendet
make: *** [envira] Fehler 2

**Comment by pfichtner** [comment, marked as answer]
Try with a linux distrubution that is not from the year 2009

**Comment by redboyhun** [comment]
Damn, I just looked at "Updated: 2024..."  😅
